common/lib/client/auth.js

```diff
@@ -260,7 +259,13 @@ var Auth = (function() {
 						}
 					}
 					cb(null, body);
-				});
+				};
+				Logger.logAction(Logger.LOG_MICRO, 'Auth.requestToken().tokenRequestCallback', 'Sending; ' + authOptions.authUrl + '; Params: ' + JSON.stringify(authParams));
+				if(authOptions.authMethod && authOptions.authMethod.toLowerCase() === 'post') {
+					Http.postUri(rest, authOptions.authUrl, authHeaders || {}, authParams, {}, authUrlRequestCallback);
```

Comment:
I thought about that too, but give all token Params are sent in the body,
this may in fact be ok. I will check what I implemented in the ruby lib
On Thu, 3 Dec 2015 at 16:18, Paddy Byers <notifications@github.com> wrote:

> In common/lib/client/auth.js
> <https://github.com/ably/ably-js/pull/173#discussion_r46572684>:
>
> > @@ -260,7 +259,13 @@ var Auth = (function() {
> >  						}
> >  					}
> >  					cb(null, body);
> > -				});
> > +				};
> > +				Logger.logAction(Logger.LOG_MICRO, 'Auth.requestToken().tokenRequestCallback', 'Sending; ' + authOptions.authUrl + '; Params: ' + JSON.stringify(authParams));
> > +				if(authOptions.authMethod && authOptions.authMethod.toLowerCase() === 'post') {
> > +					Http.postUri(rest, authOptions.authUrl, authHeaders || {}, authParams, {}, authUrlRequestCallback);
>
> If someone specifies authParams then these end up in the body and not as
> params - is that right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ably/ably-js/pull/173/files#r46572684>.
>